packages/voter-dapp/src/ResolvedRequests.js

```diff
@@ -151,7 +152,7 @@ function ResolvedRequests({ votingAccount }) {
         setResolvedEvents(nonBlacklistedRequests);
       }
     }
-  }, [allResolvedEvents, IDENTIFIER_BLACKLIST, showSpamRequests]);
+  }, [hexToUtf8, allResolvedEvents, showSpamRequests]);
```

Comment:
again recommended change by compiler

Reply:
Makes no sense...

Reply:
![image](https://user-images.githubusercontent.com/4429761/109701672-46838280-7b61-11eb-840a-02d669b39d7d.png)

Reply:
The second part confuses me @daywiss -- can you shed a light on why the blacklist "doesn't re-render the component"?

Reply:
im guessing its because this is state defined outside of a react component, so react cant react to changes to this variable. on top of that its a constant as well, so its kind of moot, it should never change.